README.md

```diff
@@ -24,12 +23,22 @@ The source code is located in `./src/boostvqe/.` and its composed of:
 * `plotscripts.py`: plotting functions.
 * `compiling_XXZ.py`: compilation for XXZ model.
 
-## How to run the code
+## Example
 
-For further information about the inputs:
+It follows a python snippet explaining how to run the boosting
 
-```sh
-python main.py --help
+```py
+
+from boostvqe.boost import main
+from boostvqe.ansatze import hdw_efficient
+
+from qibo.models.dbi.double_bracket import DoubleBracketGeneratorType
+
+help(main) #If you need some info
+
+circuit = hdw_efficient(nqubits=2, nlayers=2)
+
+main(circuit, "test_output", mode = DoubleBracketGeneratorType.group_commutator)
```

Comment:
Here I would put a comment explaining what is `test_output` and briefly also `mode`